components/places/ios/Places/Places.swift

```diff
@@ -778,9 +778,9 @@ public class PlacesWriteConnection: PlacesReadConnection {
         // This can only fail if we failed to set the `type` of the msg
         let data = try! msg.serializedData()
         let size = Int32(data.count)
-        return try data.withUnsafeBytes { (bytes: UnsafePointer<UInt8>) -> String in
+        return try data.withUnsafeBytes { bytes -> String in
```

Comment:
The current signature of `withUnsafeBytes` we are using has been [deprecated](https://developer.apple.com/documentation/foundation/data/1780450-withunsafebytes).